### PR TITLE
update dockerhub images url

### PR DESCRIPTION
**Pull request by shiywang**
Fixes: https://github.com/kubevirt/kubevirt-ansible/issues/237

@aglitke @copejon @screeley44  PTAL
Signed-off-by: Shiyang Wang <shiywang@redhat.com>

### CODE REVIEW

File: roles/cdi/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cdi_namespace: "{{ cdi_image_namespace | default('golden-images') }}"
 cdi_storageclass: "{{ cdi_kubevirt_storageclass | default('kubevirt') }}"
 apb_action: "provision"
-repo_tag: "{{ cdi_repo_tag | default('jcoperh') }}"
+repo_tag: "{{ cdi_repo_tag | default('kubevirt') }}"
 release_tag: "{{ cdi_release_tag | default('latest') }}"
```
Comment (copejon) [inline roles/cdi/defaults/main.yml]: release tags should reflect the most recent kubevirt version.  The `:latest` tag is deliberately not used.  Instead, CDI tracks kubevirt versions so as to ensure new kubevirt versions always use the most recent compatible version of CDI.

**Comment by copejon** [comment]
@shiywang Line 5 of roles/cdi/defaults/main.yml has `latest`, should have `v0.5.0-alpha.0`

**Comment by copejon** [comment]
@shiywang lgtm!

**Comment by lukas-bednar** [comment]
Thank you!

**Pull request merged**